tests/test_plot.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+import glidertools.plot as gt_plt
+
+from glidertools.load import seaglider_basestation_netCDFs
+
+
+# load some data
+filenames = "./tests/data/p542*.nc"
+
+names = ["ctd_depth", "ctd_time", "ctd_pressure", "salinity", "temperature"]
+ds_dict = seaglider_basestation_netCDFs(filenames, names, keep_global_attrs=False)
+
+dat = ds_dict["sg_data_point"]
+
+# start tests
+def test_dummy():
```

Comment:
Can you remove this one? I just put it there as a placeholder until we got real tests.